packages/@aws-cdk/aws-chatbot/README.md

```diff
@@ -34,7 +34,13 @@ const slackChannel = new chatbot.SlackChannelConfiguration(this, 'MySlackChannel
   slackChannelId: 'YOUR_SLACK_CHANNEL_ID',
 });
 
-slackChannel.addLambdaInvokeCommandPermissions();
+const invokeCommandStatement = new iam.PolicyStatement({
+  effect: iam.Effect.ALLOW,
+  resources: ["RESOURCE_ARNS_ALLOWED"],
+  actions: ["INVOKING_ACTION_ALLOWED"],
+});
```

Comment:
I think we can document this clearer and encourage a more idiomatic usage with the `grantX` methods on corresponding resources. IE, if we change this to.
```ts
const lambdaFn = new lambda.Function(this, 'MyChatbotHandler', {
  code: lambda.Code.fromAsset('./code'),
  runtime: lambda.Runtime.NODE_JS_14_X,
});
lambdaFn.grantInvoke(slackChannel.role);
```
@BLasan do you agree?

Reply:
Yuh, I think this would be ideal. But we need to add the proper permissions to the role first right?

Reply:
so `lambdaFn.grantInvoke(slackChannel.role);` should handle all of that. This should be the only grant statement needed for the chatbot to invoke the function.

Reply:
agreed